GoMeet/src/main/webapp/js/interactive-map.js

```diff
@@ -0,0 +1,77 @@
+/** Creates a map that allows users to add markers. */
+function createMap() {
+  let map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: -33.8688, lng: 151.2093}, zoom: 12}); 
+
+  map.addListener('click', (event) => {
+    createLocationForEdit(map, event.latLng.lat(), event.latLng.lng());
+  });
+  return map; 
+}
+
+/** Creates a marker that shows a textbox the user can edit. */
+function createLocationForEdit(map, lat, lng) {
+ 
+  editLocation =
```

Comment:
Same thought as above: if the map/lat/lng arguments were invalid types/values, what would happen? 
I guess this is not as important as functions in an API for example, since the code will only be called within our own application (and we are expecting valid args), but something to think about if we have time 😊

Reply:
Yup, I think that's a valid point. I was going to modify my fetchWrapper to be similar to your MeetingTimeDAO (which makes checks before sending the request) and/or doing the checks server-side and returning status based on whether the request was valid or not.